### DISCUSSION TITLE
Can we hide the "Download" option on the OpenAPI Spec?

### DISCUSSION THREAD

**Discussion by malathikrishnap** [rohit-gohri/redocusaurus#371] [Q&A] [answered]
Hi there,
Is it possible to hide the "Download" option on the spec file?
I believe this is getting added by the plugin. Why is it necessary to give this option?

<img width="391" alt="Screenshot 2024-07-02 at 6 06 37 AM" src="https://github.com/rohit-gohri/redocusaurus/assets/112460089/47da0e5b-89fa-4f85-bf1f-99496e7832a0">

**Comment by rohit-gohri** [comment, marked as answer]
Yes, you configure redoc with a `redocly.yaml` file in the root dir and specify the `hideDownloadButton` option - https://redocly.com/docs/redoc/config#hidedownloadbutton

**Reply by rohit-gohri** [reply]
See example config where I have disabled search - https://github.com/rohit-gohri/redocusaurus/blob/5d0aaf3d49245bbac64e18bd8f81222f5ab46951/website/redocly.yaml#L12-L13

**Reply by malathikrishnap** [reply]
Thank you!
